runtime/errors.go

```diff
@@ -78,6 +78,36 @@ func (e InvalidTransactionCountError) Error() string {
 	)
 }
 
+// MissingEntryPointError
+
+type MissingEntryPointError struct {
+	Expected string
+	Err      error
+}
+
+func (e *MissingEntryPointError) Unwrap() error {
+	return e.Err
+}
```

Comment:
What is the purpose of the wrapped error here? It looks like it's only ever created without an inner error: https://github.com/onflow/cadence/pull/151/files#diff-6681b213d55a5c65c5d90b211107fd52R136

Maybe just remove the `Err` field and `Unwrap` method.